di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/HomeHandler.java

```diff
@@ -21,7 +24,34 @@ public class HomeHandler {
 
                 moustacheDataModel.put("signInJourneyId", journeyId);
                 moustacheDataModel.put("uuid", userId);
-
+                moustacheDataModel.put("credentialSubjects", getCredentialSubjectData());
+                moustacheDataModel.put("evidences", getEvidencePayloads());
                 return ViewHelper.render(moustacheDataModel, "home.mustache");
             };
+
+    private static Object getCredentialSubjectData()
+            throws UnsupportedEncodingException,
+                    com.nimbusds.jose.shaded.json.parser.ParseException {
+        com.nimbusds.jose.shaded.json.JSONObject js =
```

Comment:
Could these classes be imported directrly, rather than using their fully qualified names here?